docs/Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG  BRANCH=master
+FROM quay.io/nmstate/kubernetes-nmstate-docs:${BRANCH}
+COPY Makefile Gemfile* /docs/
+RUN make -C docs install
+COPY . /docs/
```

Comment:
Why don't you copy everything into the container on line 3?

Reply:
The idea is to have a pair of layers just with ruby deps and a pair of layers just with generated webpage.

Reply:
Oh nice!